packages/app/src/modules/current.js

```diff
@@ -140,20 +140,21 @@ export const getPieCurrentFromUi = (state, action) => {
 export const getYearOverYearCurrentFromUi = (state, action) => {
     const ui = action.value
 
-    const dxItem = ui.itemsByDimension[DIMENSION_ID_DATA]
-        ? [ui.itemsByDimension[DIMENSION_ID_DATA][0]]
-        : []
+    const periodDimension = dimensionCreate(
+        DIMENSION_ID_PERIOD,
+        ui.yearOverYearCategory
+    )
+
+    const filtersWithoutPeriod = getAxesFromUi(ui).filters.filter(
+        f => f.dimension !== DIMENSION_ID_PERIOD
```

Comment:
rename `f` to `filter` or `filterItem`?

Reply:
I understand why `DIMENSION_ID_PERIOD` needs to be excluded from the `filters`, but why does `DIMENSION_ID_PERIOD` end up in `getAxesFromUi(ui).filters` to begin with? I thought the period dimension wasn't allowed on YoY in the first place?
Maybe I'm missing something, but just curious if this exclusion is unnecessary? (if period is never present anyway?)

Reply:
It can be removed I think, that filter was there already, just to be on the safe side probably.